cpp/src/utilities/column_utils.cuh

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
I don't like "hiding" this functionality in a file called "column_utils". I think you should create a file called `valid_if` (or `null_if`) if you keep that name. Put it in `src/bitmask`.